pkg/shp/cmd/build/run.go

```diff
@@ -5,72 +5,86 @@ import (
 	"fmt"
 
 	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
-	"github.com/spf13/cobra"
-
-	"k8s.io/cli-runtime/pkg/genericclioptions"
-
-	"github.com/shipwright-io/cli/pkg/shp/cmd/buildrun"
 	"github.com/shipwright-io/cli/pkg/shp/cmd/runner"
+	"github.com/shipwright-io/cli/pkg/shp/flags"
 	"github.com/shipwright-io/cli/pkg/shp/params"
 	"github.com/shipwright-io/cli/pkg/shp/resource"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-// RunCommand contains data input from user for run sub-command
+// RunCommand represents the `build run` sub-command, which creates a unique BuildRun instance to run
+// the build process, informed via arguments.
 type RunCommand struct {
-	cmd *cobra.Command
+	cmd *cobra.Command // cobra command instance
 
-	buildName string
+	buildName    string                      // build name
+	buildRunSpec *buildv1alpha1.BuildRunSpec // stores command-line flags
 }
 
-// Cmd returns cobra command object
-func (c *RunCommand) Cmd() *cobra.Command {
-	return c.cmd
-}
+const buildRunLongDesc = `
+Creates a unique BuildRun instance for informed Build (first argument), which starts the build
+process orchestrated by the Build-Controller. For example:
 
-func runCmd() runner.SubCommand {
-	runCommand := &RunCommand{
-		cmd: &cobra.Command{
-			Use:   "run <name>",
-			Short: "Start a build specified by 'name'",
-			Args:  cobra.ExactArgs(1),
-		},
-	}
+	$ shp build create my-app --source-url="..." --output-image="..."
```

Comment:
I think we can safely assume that the developer has created a Build already, therefore we don't need to reference the create command.

Reply:
Sure.